### ISSUE TITLE
Missing `verified: boolean` on UsersRecord

### ISSUE THREAD

**Issue by garlandcrow** [patmood/pocketbase-typegen#15]
I tried generating using db & json schema, but neither one has a `verified: boolean` type on the `UsersRecord`. I am using pocketbase 0.8.0-rc2 and also tested on rc1, so maybe this is something having to do w/ the changes to how users are stored in 0.8 (it seemed from the readme this would work w/ 0.8, sorry if this is isnt yet intended to be used with 0.8)

**Comment by patmood** [comment]
Ah thanks! I haven't tested this with 0.8 at all, but already know of a few things that need to change. I'll test it myself soon but you can list any more issues you find in this thread and I'll fix them up.

I don't think it'll be backwards compatible because of some type changes, so users will need to use a version that matches their PB version.

**Comment by karniv00l** [comment]
Let me also share my current diff after typegen (I'm running Pocketbase RC branch):

```ts
export type BaseRecord = {
        id: RecordIdString
        created: IsoDateString
        updated: IsoDateString
-  collectionId: string
-  collectionName: string
-  expand?: { [key: string]: any }
+       "@collectionId": string
+       "@collectionName": string
+       "@expand"?: { [key: string]: any }
 }

...
 export type UsersRecord = {
        avatar?: string
-  username: string
-  email: string
-  verified: boolean
 }
```

(tabs vs spaces and single vs double quotes are not a big deal for me since it can be auto-fixed locally)

**Comment by patmood** [comment]
Looks like 0.8 has 2 different types of collections now: Base and Auth.

Auth collections have these system fields: `id` , `created` , `updated` , `username` , `email` , `emailVisibility` , `verified`

So `UsersResponse` will be an intersection with `AuthRecord` instead of `BaseRecord`. I'll probably rename them though because you can make your own collection called "base" the types will conflict.

I'll publish an update soon. There's a few other backwards incompatible changes I want to include

**Comment by patmood** [comment]
Fixed in v1.1.0, which should be used with PocketBase v0.8.x 🎉
